File: app/src/main/java/nl/rsdt/japp/jotial/maps/management/controllers/BravoVosController.java
package nl.rsdt.japp.jotial.maps.management.controllers;

/**
 * @author Dingenis Sieger Sinke
 * @version 1.0
 * @since 31-7-2016
 * Description...
 */


import java.util.ArrayList;

import nl.rsdt.japp.jotial.data.structures.area348.BaseInfo;

/**
 * @author Dingenis Sieger Sinke
 * @version 1.0
 * @since 31-7-2016
 * Description...
 */
public class BravoVosController extends VosController {

    public static final String CONTROLLER_ID = "BravoVosController";

    public static final String STORAGE_ID = "STORAGE_VOS_B";

    public static final String BUNDLE_ID = "VOS_B";

    public static final String REQUEST_ID = "REQUEST_VOS_B";

    public BravoVosController() {

    }

    @Override
    public String getTeam() {
        return "b";
    }

    @Override
    public String getId() {
        return CONTROLLER_ID;
    }

    @Override
    public String getStorageId() {
        return STORAGE_ID;
    }

    @Override
    public String getBundleId() {
        return BUNDLE_ID;
    }

    @Override
    public ArrayList<BaseInfo> searchFor(String query) {
        return super.searchFor(query);
    }
}

File: app/src/main/java/nl/rsdt/japp/jotial/maps/management/controllers/BravoVosController.kt
package nl.rsdt.japp.jotial.maps.management.controllers

import android.os.Bundle
import nl.rsdt.japp.jotial.maps.wrapper.ICircle
import java.util.ArrayList

/**
 * @author Dingenis Sieger Sinke
 * @version 1.0
 * @since 31-7-2016
 * Description...
 */

/**
 * @author Dingenis Sieger Sinke
 * @version 1.0
 * @since 31-7-2016
 * Description...
 */
class BravoVosController() : VosController() {


    override val team: String
        get() = "b"

    override val id: String
        get() = CONTROLLER_ID

    override val storageId: String
        get() = STORAGE_ID

    override val bundleId: String
        get() = BUNDLE_ID

    companion object {

        val CONTROLLER_ID = "BravoVosController"

        val STORAGE_ID = "STORAGE_VOS_B"

        val BUNDLE_ID = "VOS_B"

        val REQUEST_ID = "REQUEST_VOS_B"
    }

}

